core/loader/utils.ts

```diff
@@ -1,11 +1,19 @@
 import type { TextResponse } from '../download.js'
 import type { PreviewCandidate } from '../preview.js'
 
+export const HREF_PATTERNS = {
+  atom: /feed\.|\.atom|\/atom/i,
+  rss: /feed\.|\.rss|\/rss/i
```

Comment:
You don’t need to keep `feed.` in both RegExps. Let’s only Atom look for `feed.`